t/e2e/jail/jail_rpc_test.go

```diff
@@ -110,138 +101,6 @@ func (s *JailRPCTestSuite) TestRegressionGetTransactionReceipt() {
 	expected := `{"jsonrpc":"2.0","id":7,"result":null}`
 	s.Equal(expected, got)
 }
-func (s *JailRPCTestSuite) TestContractDeploymentLES() {
-	s.testContractDeployment(false)
-}
-func (s *JailRPCTestSuite) TestContractDeploymentRPC() {
-	s.testContractDeployment(true)
-}
-
-func (s *JailRPCTestSuite) testContractDeployment(upstream bool) {
```

Comment:
Not sure if I should be removing these TestContractDeploymentRPC tests. Presumably contract creation should be called very differently via RPC after the changes in interface. Looking for input on how to do this.

Reply:
@mandrigin @flexsurfer I'm still confused as to whether this test should be removed. The test runs `web3.eth.contract.new()` in order to create the contract. This in turn calls the sendTransaction handler that I removed from backend.registerHandlers. 

What is the correct way to re-write this test? Whats the correct way to run a cell without calling backend.SendTransaction?

Other than that, this PR should be ready.

Reply:
The current approach should not allow calling `eth_sendTransaction` at all. If such an RPC call is received, we should return an error as it should be handled on status-react side by calling `SendTransaction()` function from `library.go`. I think this should be tested here. Similarly with `personal_*` APIs.

`backend.SendTransaction` uses some struct to send transactions or create contracts and only that struct should be unit tested. `backend.SendTransaction` can also have some unit/integration tests but I have seen mostly it covered.